src/main/java/com/dev/cinema/model/dto/UserRequestDto.java

```diff
@@ -0,0 +1,29 @@
+package com.dev.cinema.model.dto;
+
+public class UserRequestDto {
+
```

Comment:
```suggestion
```
remove these empty line everywhere in your classes

Reply:
done